deps/uv/uv.gyp

```diff
@@ -263,6 +263,7 @@
             'src/unix/random-getrandom.c',
             'src/unix/random-sysctl-linux.c',
             'src/unix/sysinfo-loadavg.c',
+            'src/unix/random-getentropy.c'
```

Comment:
It looks like we were using trailing commas before. Would you mind adding a comma to this new line.

Reply:
Thanks for quick reviewing, I have updated :)